worlds/hk/docs/en_Hollow Knight.md

```diff
@@ -20,3 +20,10 @@ When the Hollow Knight player picks up an item from a location and it is an item
 player's recent items display as an item being sent to another player. If the item is for another Hollow Knight player
 then the sprite will be that of the item's original sprite. If the item belongs to a player that is not playing Hollow
 Knight then the sprite will be the Archipelago logo.
+
+## What logic does the randomization rely on?
+
+Randomization may expect you to use the Benchwarp mod to return to the starting location. Warping elsewhere will never be required to complete the game. For more details, see the documentation for the Hollow Knight randomizer mod, below:
```

Comment:
This almost makes it sound like you're expected not to benchwarp anywhere other than start which isn't really true - maybe a ", though warping to other benches you have been to already is expected for convenience" or something along those lines